api/src/app.controller.ts

```diff
@@ -25,14 +26,55 @@ export class AppController {
     return this.appService.getGameState();
   }
 
-  @Get('marbleplayable')
-  isMarblePlayable(): Boolean {
-    // return this.appService.getIsMarblePlayable();
-    return false;
+  @Get('restartgame')
+  getRestartGame(): GameState {
+    return this.appService.getRestartGame();
+  }
+
+  @Post('gamestatehaschanged')
+  getGameStateHasChanged(@Body() body): boolean {
```

Comment:
This should probably be a `get`, not a `post`

Reply:
That's what i thought at first, thanks :)